webtasks/cron_mongodb.js

```diff
@@ -0,0 +1,230 @@
+var _ = require('lodash');
+var Bluebird = require('bluebird');
+
+var mongo;
+
+
+var webtask = function (context, req, res) {
+    var action = validate_method({
+        put_job: 'PUT',
+        get_job: 'GET',
+        destroy_job: 'DELETE',
+        list_jobs: 'GET',
+        job_history: 'GET',
+        reserve_jobs: 'POST',
+        renew_reservation: 'PUT',
+        release_reservation: 'DELETE',
+    });
```

Comment:
The compiler is probably smart enough, but this `action` object can also be moved outside of this function.